cmd/utils.go

```diff
@@ -0,0 +1,63 @@
+package main
```

Comment:
rename the file logsProcess.go ? as this is exactly what it does

Reply:
Renamed the file to `logsProcess`